web/src/components/Settings/DataStore/DataStore.tsx

```diff
@@ -24,6 +24,11 @@ const DataStore = ({config}: IProps) => {
   return (
     <S.Wrapper data-cy="config-datastore-form">
       <S.FormContainer>
+        <S.Description>
+          Tracetest needs configuration information to be able to retrieve your trace from your distributed tracing
+          solution. Select your tracing data store and enter the configuration info.
+        </S.Description>
+        <S.Title>Choose type of OTEL data store</S.Title>
```

Comment:
This sounds weird. Maybe `Choose your OTel data store`

Reply:
I would use the full name `OpenTelemetry`.

> Choose OpenTelemetry data store